packages/animations/lib/src/open_container.dart

```diff
@@ -235,21 +235,21 @@ class _OpenContainerState extends State<OpenContainer> {
   final GlobalKey _closedBuilderKey = GlobalKey();
 
   Future<void> openContainer() async {
-    await Navigator.of(context, rootNavigator: widget.useRootNavigator)
-        .push(_OpenContainerRoute(
-      closedColor: widget.closedColor,
-      openColor: widget.openColor,
-      closedElevation: widget.closedElevation,
-      openElevation: widget.openElevation,
-      closedShape: widget.closedShape,
-      openShape: widget.openShape,
-      closedBuilder: widget.closedBuilder,
-      openBuilder: widget.openBuilder,
-      hideableKey: _hideableKey,
-      closedBuilderKey: _closedBuilderKey,
-      transitionDuration: widget.transitionDuration,
-      transitionType: widget.transitionType,
-    ));
+    await Navigator.of(context, rootNavigator: widget.useRootNavigator).push(
+        _OpenContainerRoute(
+            closedColor: widget.closedColor,
+            openColor: widget.openColor,
+            closedElevation: widget.closedElevation,
+            openElevation: widget.openElevation,
+            closedShape: widget.closedShape,
+            openShape: widget.openShape,
+            closedBuilder: widget.closedBuilder,
+            openBuilder: widget.openBuilder,
+            hideableKey: _hideableKey,
+            closedBuilderKey: _closedBuilderKey,
+            transitionDuration: widget.transitionDuration,
+            transitionType: widget.transitionType,
+            useRootNavigator: widget.useRootNavigator));
```

Comment:
nit: use a trailing comma so that the formatter doesn't reformat the entire block
```suggestion
            useRootNavigator: widget.useRootNavigator,));
```